web-client/src/presenter/actions/CaseInventoryReport/getCaseInventoryReportAction.ts

```diff
@@ -10,21 +10,26 @@ import { state } from '@web-client/presenter/app.cerebral';
 export const getCaseInventoryReportAction = async ({
   applicationContext,
   get,
+  props,
   store,
-}: ActionProps) => {
+}: ActionProps<{ selectedPage: number }>) => {
   const { associatedJudge, status } = get(state.screenMetadata);
+  const { selectedPage } = props;
 
   if (associatedJudge || status) {
     const reportData = await applicationContext
       .getUseCases()
       .getCaseInventoryReportInteractor(applicationContext, {
         associatedJudge,
+        selectedPage,
         status,
       });
 
-    store.set(state.caseInventoryReportData, {
-      foundCases: reportData.foundCases,
-    });
+    store.set(state.caseInventoryReportData.foundCases, reportData.foundCases);
+    store.set(
+      state.caseInventoryReportData.foundCasesCount,
+      reportData.totalCount,
+    );
```

Comment:
Not a big deal either way, but is there any particular reason we want to split apart `foundCases` and `foundCasesCount`? The latter seems redundant given that the time complexity of `Array.length` is O(1). (I noticed this because some tests look strange when we say `foundCases: []` but `foundCasesCount: 3`, etc.)

Reply:
Great point! I think unclear naming conventions are the culprit:

`foundCases` and `foundCasesCount` are tracked separately in state because the former actually contains the _current page's_ found cases, while the latter represents the total number of found cases _across all pages_.

`foundCases` should really be something like `foundCasesForCurrentPage`, while `foundCasesCount` should be something like `foundCasesTotalCount`. I'll clean this up.

Reply:
Ah, that makes sense. Thanks!